multipaz/build.gradle.kts

```diff
@@ -161,7 +158,8 @@ kotlin {
                 implementation(libs.bouncy.castle.bcprov)
                 implementation(libs.bouncy.castle.bcpkix)
                 implementation(project(":multipaz-doctypes"))
-                implementation(project(":multipaz:matcherTest"))
+                implementation(project(":multipaz-dcapi"))
+                implementation(project(":multipaz-dcapi:matcherTest"))
```

Comment:
I think it would be better to move the tests of the DC implementations to multipaz-dcapi than leaving them in the core Multipaz library.